### ISSUE TITLE
suggestion: mime

### ISSUE THREAD

**Issue by richajak** [talentlessguy/es-mime-types#2]
Hi @talentlessguy thanks for all the hard work done with tinyhttp  I have a suggestion, should the 'mime-db' dependency be replaced with something lighter and still able to handle majority of file extensions, such as mime/lite or mrmime  ?

https://github.com/broofa/mime/lite
https://github.com/lukeed/mrmime

Thanks

**Comment by v1rtl** [comment]
mrmime looks good, I'll try to replace es-mime-type with that

**Comment by v1rtl** [comment]
@richajak this is currently blocked by https://github.com/lukeed/mrmime/issues/2

**Comment by psulek** [comment]
@talentlessguy cant u just use that `charset` function you mention directly in your package? 
Because using **es-mime-types** depending on file `mime-db\db.json` which is ~180kb. 
Thats hell too much.

**Comment by v1rtl** [comment]
@psulek yeah I'm going to use `mrmime` with custom charset func, working on it now

**Comment by v1rtl** [comment]
@psulek mrmime has a too different api, same with mime... could you please create a pull request using either of those?

**Comment by psulek** [comment]
I can look at it tomorrow
